librz/main/rz-asm.c

```diff
@@ -166,49 +170,129 @@ static int show_analinfo(RzAsmState *as, const char *arg, ut64 offset) {
+		" -q",
+		"",
+		"    quiet mode",
+		" -r",
+		"",
+		"    output in rizin commands",
+		" -s",
+		"[syntax]",
+		"   Select syntax (intel, att)",
+		" -v",
+		"",
+		"    Show version information",
+		" -x",
+		"",
+		"    Use hex dwords instead of hex pairs when assembling.",
+		" -w",
+		"",
+		"    What's this instruction for? describe opcode",
+	};
 	if (v != 1) {
-		printf(" -a [arch]    Set architecture to assemble/disassemble (see -L)\n"
-		       " -A           Show Analysis information from given hexpairs\n"
-		       " -b [bits]    Set cpu register size (8, 16, 32, 64) (RZ_ASM_BITS)\n"
-		       " -B           Binary input/output (-l is mandatory for binary input)\n"
-		       " -c [cpu]     Select specific CPU (depends on arch)\n"
-		       " -C           Output in C format\n"
-		       " -d, -D       Disassemble from hexpair bytes (-D show hexpairs)\n"
-		       " -e           Use big endian instead of little endian\n"
-		       " -I           Display lifted RzIL code (same input as in -d, IL is also validated)\n"
-		       " -E           Display ESIL expression (same input as in -d)\n"
-		       " -f [file]    Read data from file\n"
-		       " -F [in:out]  Specify input and/or output filters (att2intel, x86.pseudo, ...)\n"
-		       " -h, -hh      Show this help, -hh for long\n"
-		       " -i [len]     ignore/skip N bytes of the input buffer\n"
-		       " -j           output in json format\n"
-		       " -k [kernel]  Select operating system (linux, windows, darwin, ..)\n"
-		       " -l [len]     Input/Output length\n"
-		       " -L           List Asm plugins: (a=asm, d=disasm, A=analyze, e=ESIL)\n"
-		       " -o,-@ [addr] Set start address for code (default 0)\n"
-		       " -O [file]    Output file name (rz-asm -Bf a.asm -O a)\n"
-		       " -p           Run SPP over input for assembly\n"
-		       " -q           quiet mode\n"
-		       " -r           output in rizin commands\n"
-		       " -s [syntax]  Select syntax (intel, att)\n"
-		       " -v           Show version information\n"
-		       " -x           Use hex dwords instead of hex pairs when assembling.\n"
-		       " -w           What's this instruction for? describe opcode\n"
-		       " If '-l' value is greater than output length, output is padded with nops\n"
-		       " If the last argument is '-' reads from stdin\n");
-		printf("Environment:\n"
-		       " RZ_NOPLUGINS      do not load shared plugins (speedup loading)\n"
-		       " RZ_ASM_ARCH       same as rz-asm -a\n"
-		       " RZ_ASM_BITS       same as rz-asm -b\n"
-		       " RZ_DEBUG          if defined, show error messages and crash signal\n"
-		       "");
+		for (int i = 0; i < sizeof(options) / sizeof(options[0]); i += 3) {
+			if (i + 1 < sizeof(options) / sizeof(options[0])) {
+				printOption(options[i], options[i + 1], options[i + 2]);
+			}
+		}
 	}
+	printf("%s%s", COLOR_RESET, "  If '-l' value is greater than output length, output is padded with nops\n\
```

Comment:
Please concat strings with `"bla" "foo"` like before instead of using `\n` to preserve nicer formatting in the source code